.github/workflows/request-reviewer.yml

```
@@ -24,6 +24,14 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.2
+      - name: Checkout membership data from main branch
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.2
+        with:
+          ref: main
+          path: .
+          sparse-checkout: |
+            .ci/magician/github/membership_data.go
+          sparse-checkout-cone-mode: false
```

```suggestion
        with:
          ref: main
```

Based on discussion, I think we just want to checkout everything from main.

See https://github.com/trodge/magic-modules/pull/14
https://github.com/trodge/magic-modules/actions/runs/10725729173/job/29744244495

Feel free to open a PR with trodge/magic-modules as base if you can think of any edge cases I didn't test.